commands/sdcli_go_integration

```diff
@@ -16,10 +16,17 @@ fi
 
 mkdir -p .coverage
 
-INTTESTS=$(go test -tags=integration -list . ./tests)
+PKGS="$(go list ./... | paste -sd "," -)"
+if [[ -f "${PWD}/main.go" ]]; then
+    PKGS="$(go list ./... | sed 1d | paste -sd "," -)"
+fi
+
+exec 5>&1
+INTTESTS=$(go test -v -tags=integration -cover -coverpkg="${PKGS}" -coverprofile=.coverage/integration.cover.out -race ./tests | tee >(cat - >&5))
 FAIL="$(echo "${INTTESTS}" | grep 'FAIL')"
 if [[ ${FAIL} != "" ]]; then
-    echo "A setup or compilation failure occurred."
+    echo "A setup or compilation failure occurred:"
+    echo "${INTTESTS}"
```

Comment:
question: Why are we both echoing the output here and redirecting the subshell to stdout? Seems like it should be one or the other.

Reply:
oh good catch.  No need to echo if it's already printed.  I'll remove that echo.